ts/features/mvl/screens/details/components/attachment/__test__/MvlAttachments.test.tsx

```diff
@@ -38,18 +47,36 @@ describe("MvlAttachments", () => {
         const res = renderComponent({
           attachments: [mvlMockPdfAttachment, mvlMockOtherAttachment]
         });
-        expect(res.queryByText(mvlMockPdfAttachment.name)).not.toBeNull();
         expect(
-          res.queryByText(formatByte(mvlMockPdfAttachment.size))
+          res.queryByText(mvlMockPdfAttachment.displayName)
         ).not.toBeNull();
-        expect(res.queryByText(mvlMockOtherAttachment.name)).not.toBeNull();
+        if (mvlMockPdfAttachment.size) {
+          expect(
+            res.queryByText(formatByte(mvlMockPdfAttachment.size))
+          ).not.toBeNull();
+        }
         expect(
-          res.queryByText(formatByte(mvlMockOtherAttachment.size))
+          res.queryByText(mvlMockOtherAttachment.displayName)
         ).not.toBeNull();
+        if (mvlMockOtherAttachment.size) {
```

Comment:
I suggest we avoid conditionals in unit tests: by branching, we test more than one thing at a time. Not mandatory for this review.

Reply:
In this case is only required for typescript type inference (I want to avoid the `.!` operator as much as possible). Btw enforced with other assertions in 71a8e170169fa3a0d94a52d52b8b9068bc230911 in order to make the test fails if the condition is not verified.